hardhat.config.js

```diff
@@ -65,8 +65,8 @@ module.exports = {
   networks: {
     hardhat: {
       allowUnlimitedContractSize: true,
-      // forking: {
```

Comment:
could you please keep the `forking` block commented? in this way unit tests will pass

Reply:
I using mainnet fork for testing the IdleMstableStrategy. 
If not forked, we would need to deploy MStable, Uniswapv2 locally. 

Any other solution, please suggest.

Reply:
you can use it locally no problem on that, just don't commit it so everything will work as outlined here https://github.com/Idle-Labs/idle-tranches#testing